integration/spark/src/main/scala/org/apache/spark/sql/CarbonSqlParser.scala

```diff
@@ -412,24 +413,34 @@ class CarbonSqlParser()
             case Token("TOK_LIKETABLE", child :: Nil) =>
               likeTableName = child.getChild(0).getText()
 
+            case Token("TOK_QUERY", _) =>
+              isCreateAsSelect = true
+
             case _ => // Unsupport features
           }
 
-
-          // validate tblProperties
-          if (!CommonUtil.validateTblProperties(tableProperties, fields)) {
-            throw new MalformedCarbonCommandException("Invalid table properties")
+          if (isCreateAsSelect) {
+            CreateCarbonTableAsSelect(
```

Comment:
There is no table property for CreateTableAsSelect? How if user want to exclude some dictionary?